landing/src/index.css

```
@@ -0,0 +1,584 @@
+.demo-copy-btn{position:absolute;top:8px;right:8px;opacity:0;transition:opacity 0.15s;background:transparent;border:1px solid var(--border);color:var(--text3);font-size:10px;padding:3px 8px;border-radius:4px;cursor:pointer;font-family:var(--mono)}
+.demo-attack-btn:hover .demo-copy-btn{opacity:1}
+.demo-copy-btn:hover{color:var(--cyan);border-color:var(--cyan)}
+
+.demo-panel{background:var(--surface);border:1px solid var(--border);border-radius:var(--radius);overflow:hidden;min-height:360px}
+.demo-panel-header{background:var(--surface2);border-bottom:1px solid var(--border);padding:12px 18px;display:flex;align-items:center;gap:8px}
+.demo-panel-dot{width:10px;height:10px;border-radius:50%}
+.demo-panel-title{font-size:11px;color:var(--text3);font-family:var(--mono)}
+.demo-panel-body{padding:24px}
+
+.demo-prompt-label{font-size:10px;font-family:var(--mono);color:var(--text3);letter-spacing:0.1em;text-transform:uppercase;margin-bottom:8px}
+.demo-prompt-text{font-size:13px;color:var(--text);line-height:1.7;padding:14px;background:var(--bg);border:1px solid var(--border);border-radius:8px;margin-bottom:20px;font-family:var(--mono);font-size:11.5px}
```

Stylelint (17.12.0)

[error] Duplicate property "font-size" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)